src/Stream.ts

```diff
@@ -25,17 +25,21 @@ export default class Stream<T> {
 
   unsubscribeFn?: UnsubscribeFn
 
+  onConnectionLostCallback?: OnConnectionLostCallback
+
   constructor(
     client: Client,
     topics: string[],
     decoder: MessageDecoder<T>,
-    contentTopicUpdater?: ContentTopicUpdater<T>
+    contentTopicUpdater?: ContentTopicUpdater<T>,
+    onConnectionLostCallback?: OnConnectionLostCallback
```

Comment:
my only nit is that we should name this argument `onConnectionLost` across the board instead of `onConnectionLostCallback`. that being said, i'm more interested in consistency than anything. there's some mixing and matching between `onConnectionLost` and `onConnectionLostCallback` (excluding the type name, which i think is fine).

Reply:
Fair point. Fixed.